bin/data_frame_analyzer/Makefile

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+include $(CPP_SRC_HOME)/mk/defines.mk
+
+TARGET=$(OBJS_DIR)/data_frame_analyzer$(EXE_EXT)
+
+INSTALL_DIR=$(CPP_PLATFORM_HOME)/bin
+
+ML_LIBS=$(LIB_ML_CORE) $(LIB_ML_API)
```

Comment:
I suspect that this will create link issues on linux. You'll probably find you need to add in $(LIB_ML_MODEL) as a minimum.